### ISSUE TITLE
Add curated content on "Good Enough Practices in Scientific Computing

### ISSUE THREAD

**Issue by pagrubel** [betterscientificsoftware/betterscientificsoftware.github.io#340]
Good Enough Practices in Scientific Computing is a follow on article to previous Best Practices
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005510 
I plan to write up an article reviewing the above article similiar to the previous on for Best Practices

**Comment by bartlettroscoe** [comment]
@bartlettroscoe and @JeanShuler have agreed to review.

**Comment by bartlettroscoe** [comment]
@bartlettroscoe and @JeanShuler have agreed to review.

**Comment by JeanShuler** [comment]
I reviewed this article and think it is valuable to all levels of code developers and researchers

**Comment by bartlettroscoe** [comment]
Should this be closed yet?  I did not look at this yet.  If a single "okay" is enough, then that is fine.

**Comment by JeanShuler** [comment]
Sorry Ross – I clicked the wrong button!
Jean

**Comment by JeanShuler** [comment]
issue reopened!

**Comment by pagrubel** [comment]
I guess when Jean closed this it and reopened it, the issue was move back to the idea backlog. I'm moving it to topic review and another reviewer @bartlettroscoe or anyone else approves it, then it can be moved to Development Backlog. (in reality this article has already been approved. We are just moving it through the board to see how the process works.

**Comment by pagrubel** [comment]
This seems to be a bottleneck, should we assign people or will we have people that are looking at the board?

**Comment by bartlettroscoe** [comment]
Does a sign-off require reading the entire publication or can we just browse it and give our opinion?

**Comment by markcmiller86** [comment]
In some google doc somewhere, I had proposed that when we give our opinion, we also give some indication of how thorough our review was. I kinda sorta see two or three *tiers* here...

- Just found this article...haven't read it but thought it might warrant a closer look
- I've done a _cursory review_ it (abstract, intro, conclusions, maybe some random paragraphs in the middle) and it looks promising (or not).
- I've given this a _thorough read through_, end-to-end and recommend it (or not)

**Comment by markcmiller86** [comment]
BTW...my comments above are *in general* and not intended to be specific to issue these comments are attached to.

**Comment by pagrubel** [comment]
Good question. In this case you can just skim it, because it was actually already approved. I'm just going through the process of using the board.

**Comment by bartlettroscoe** [comment]
I read over a good bit of the material in:

* https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005510

I think it is good advice but I think that perhaps not appropriate for ECP projects (who are the main customer for the IDEAS-ECP project).  These "good enough" practices are **NOT** good enough for any ECP project that hopes to live past the end of ECP (or even get to the end of ECP).  All ECP projects must use a version control system (actually, git is mandated due to the GitLab CI infrastructure being put together).  Also, all ECP projects must write strong automated tests.  That is not part of these "good enough" practices (all they mention is "Provide a simple example or test data set (2i)" and only hit at automated testing and that is not "good enough" for ECP).

My option is that this article is meant to appeal to a set of people and projects outside of the scope of a serious software-based project like ECP.  I would hate for an ECP team to find this article and then only do what is listed here and say "hey, we did what was recommended in the 'good enough' practices advocated by the IDEAS-ECP team so why is this not enough?"

If we do provide a curated content article on this, we need to provide a disclaimer that this is not "good enough" for many serious CSE projects like any ECP project but might be of some help for other less software-focused projects that are not even doing this.

**Comment by pagrubel** [comment]
Although I added a milestone to this, I think we determined that milestones won't work well for this level. I suggest we need another column on the project board for those items planned for the current month with the deadline needed for that month. Also, I moved this because it was approved prior to this process. Thanks for the review @bartlettroscoe I will definitely put consider your cautions while writing the review.
